### ISSUE TITLE
Add build/ dependency-reduced-pom.xml and scylla-apiclient/target/ to .gitignore

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla-jmx#123]
Now that scylla-jmx became a submodule of scylla, I see the following artifacts under scylla tools/jmx after ninja-build
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	build/
	dependency-reduced-pom.xml
	scylla-apiclient/target/
```

These need to be added to `.gitignore`  otherwise git thinks that the submodule is dirty.

**Comment by bhalevy** [comment]
Cc @penberg

**Comment by penberg** [comment]
@bhalevy All of the directories are in `.gitignore`, no?

https://github.com/scylladb/scylla-jmx/blob/master/.gitignore

I don't quite understan why they're not getting picked up...

**Comment by bhalevy** [comment]
I don't see the problem anymore.  Guess this issue can be closed with c0d9d0f05174638ef13c4807906941364f5627ac
